CHANGES.rst

```diff
@@ -22,7 +22,3 @@ Features
 - Download a package with its dependencies
   `#6004 <https://pulp.plan.io/issues/6004>`_
 
-
-----
```

Comment:
@daviddavis @dralley @mdellweg towncrier adds `----` at the end which breaks sphinx, the first release needs some sphinx tweaks